__tests__/SortFilter.test.ts

```diff
@@ -8,7 +8,7 @@ describe("SortFilter", () => {
       property: "sort",
       by: SortDirection.NONE,
     });
-    expect(parser.fromQuery(query.build())).toEqual({});
+    expect(parser.fromQuery(query.build())).toEqual(undefiend);
```

Comment:
@emrekeskinmac is this a new type ? `undefiend` ? :)

Reply:
@cyilcode omg :D sorry